config/webhook/manifests.yaml

```diff
@@ -474,6 +474,27 @@ webhooks:
     resources:
     - imagepulljobs
   sideEffects: None
+- admissionReviewVersions:
```

Comment:
add vingress.kb.io and vservice.kb.io in [file](https://github.com/openkruise/kruise/blob/master/config/webhook/patch_manifests.yaml)